src/TestFramework/DataFaker.php

```diff
@@ -1145,6 +1145,20 @@ public function applePayTransactionReference()
         return $result;
     }
 
+    /**
+     * Return a path.
```

Comment:
What kind of path? What is this path for?

Reply:
This is to fake a path to a file. For my case I need this to test getting and setting pem/key certs. I'm making it general so that it can be used for a different context in the future.

I used it here: https://github.com/vimeo/omnipay-vindicia/pull/42/files#diff-fa49bbd6059ef71c54689eb53f1b3f1eR18

Reply:
I can update the doc block to make it more clear.